generators/client/templates/src/main/webapp/app/blocks/config/_translation.storage.js

```diff
@@ -0,0 +1,28 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('<%=angularAppName%>')
+        .factory('translationStorage', translationStorage);
```

Comment:
Yes I guess, coz this acts as a service provider  for translate config rite
On 4 Mar 2016 14:13, "Anders Steiner" <notifications@github.com> wrote:

> In
> generators/client/templates/src/main/webapp/app/blocks/config/_translation.storage.js
> <https://github.com/jhipster/generator-jhipster/pull/3073#discussion_r54993754>
> :
>
> > @@ -0,0 +1,28 @@
> > +(function() {
> > +    'use strict';
> > +
> > +    angular
> > +        .module('<%=angularAppName%>')
> > +        .factory('translationStorage', translationStorage);
>
> should this be called translationStorageProvider as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/3073/files#r54993754>
> .
>